recipes-nodes/render-config/fetch-config.sh

```diff
@@ -13,7 +13,7 @@ set -e
 
 INIT_CONFIG_FILE="/etc/init-config.json"
 TEMPLATED_CONFIG_FILES="/etc/td-agent-bit/td-agent-bit.conf /etc/process-exporter/process-exporter.yaml /etc/prometheus/prometheus.yml /etc/rbuilder.config /etc/rclone.conf /etc/orderflow-proxy.conf /etc/system-api/systemapi-config.toml /etc/rbuilder-bidding/rbuilder-bidding-token"
-TEMPLATED_CONFIG_FILES_UNSAFE="/etc/rbuilder-bidding/bidding-service.toml"
+TEMPLATED_CONFIG_FILES_UNSAFE="/etc/rbuilder-bidding/bidding-service.toml /etc/disk-encryption/key"
```

Comment:
Unsafe because file may contain special symbols that could be otherwise be escaped in the safe version

Reply:
what happens if the /etc/disk-encryption/key.manifest is not there in case the image was built with tpm approach?
Would that cause this script to fail, or would it simply continue with the rest of the templates without failing?

Reply:
Added a function that checks if template is optional.